src/YoTennis/Models/PlayerExtentions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace YoTennis.Models
+{
+    public static class PlayerExtentions
```

Comment:
You can just call it `Extensions` and use for `Player`, `PlayerScore` and other extensions.